### PR TITLE
Add working example to quickstart section of README

### LINKED ISSUE #2364: Add a quick example in the quickstart section of the README file?
Currently, the [Quickstart](https://github.com/GenericMappingTools/pygmt#quickstart) section in the README file only have one subsection with quick installation instructions. I think a real quickstart section should also have a simple working example so that users have a quick idea about how PyGMT works. So I propose to add a simple example in the quickstart section, like:

```
import pygmt
fig = pygmt.Figure()
fig.coast(projection="H10c", region="g", frame=True, land="gray")
fig.show()
```

### PR DESCRIPTION
**Pull request by yvonnefroehlich**
**Description of proposed changes**

This PR aims to add a simple working example to the Quickstart section in the README file.

Fixes #2364

**Reminders**

- [ ] Run `make format` and `make check` to make sure the code follows the style guide.
- [ ] Add tests for new features or tests that would have caught the bug that you're fixing.
- [ ] Add new public functions/methods/classes to `doc/api/index.rst`.
- [ ] Write detailed docstrings for all functions/methods.
- [ ] If wrapping a new module, open a 'Wrap new GMT module' issue and submit reasonably-sized PRs.
- [ ] If adding new functionality, add an example to docstrings or tutorials.
- [ ] Use underscores (not hyphens) in names of Python files and directories.

**Slash Commands**

You can write slash commands (`/command`) in the first line of a comment to perform
specific operations. Supported slash commands are:

- `/format`: automatically format and lint the code
- `/test-gmt-dev`: run full tests on the latest GMT development version

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -102,6 +102,19 @@ If you use `mamba <https://mamba.readthedocs.org/>`__::
 
 For other ways to install ``pygmt``, see `full installation instructions <https://www.pygmt.org/latest/install.html>`__.
 
+Getting started
++++++++++++++++
+
+As a starting point you can try the following example::
```
Comment (seisman) [inline README.rst:108]:
```suggestion
As a starting point, you can open a Python console or a Jupyter Notebook, and try 
the following example::
```

Comment (yvonnefroehlich) [reply README.rst]: It's only a minor detail, but what is prefered "Jupyter Notebook" (upper-case) or "Jupter notebook" (lower-case)?
Currently, it seems we are mostly using the lower-case version:
```
$ grep "Jupyter" */*
grep: ci/requirements: Is a directory
grep: doc/_static: Is a directory
grep: doc/_templates: Is a directory
grep: doc/api: Is a directory
doc/changes.md:* Add instructions to install pygmt kernel for Jupyter users ([#2153](https://github.com/GenericMappingTools/pygmt/pull/2153))
doc/changes.md:* Support tab auto-completion in Jupyter ([#1282](https://github.com/GenericMappingTools/pygmt/pull/1282))
doc/changes.md:* Improve Figure.show for displaying previews in Jupyter notebooks and external viewers ([#529](https://github.com/GenericMappingTools/pygmt/pull/529))
doc/contributing.md:must be formatted manually. To play nicely with Jupyter and IPython, **keep docstrings
doc/install.rst:interpreter (e.g., in a Jupyter notebook)::
doc/install.rst:* `IPython <https://ipython.org>`__: For embedding the figures in Jupyter notebooks (recommended).
doc/install.rst:Notes for Jupyter users
doc/install.rst:get a ``ModuleNotFoundError`` when importing pygmt in Jupyter, you may need to
doc/install.rst:After that, you need to restart Jupyter, open your notebook, select the
doc/team.md:C API, supporting rich display in Jupyter notebooks, and integrating with
grep: examples/gallery: Is a directory
grep: examples/get_started: Is a directory
grep: examples/projections: Is a directory
grep: examples/tutorials: Is a directory
pygmt/__init__.py:ecosystem as well as support for rich display in Jupyter notebooks.
grep: pygmt/clib: Is a directory
grep: pygmt/datasets: Is a directory
pygmt/figure.py:# Show figures in Jupyter notebooks if available
pygmt/figure.py:    if get_ipython and "IPKernelApp" in get_ipython.config:  # Jupyter Notebook enabled
pygmt/figure.py:        Inserts the preview in the Jupyter notebook output if available,
pygmt/figure.py:            The image resolution (dots per inch) in Jupyter notebooks.
pygmt/figure.py:            The image width (in pixels) in Jupyter notebooks.
pygmt/figure.py:            - **notebook**: PNG preview [Default in Jupyter notebooks]
pygmt/figure.py:                        "or run the script in a Jupyter notebook."
pygmt/figure.py:        For the Jupyter notebook or IPython Qt console.
pygmt/figure.py:        - **notebook**: PNG preview [Default in Jupyter notebooks]
grep: pygmt/helpers: Is a directory
grep: pygmt/src: Is a directory
grep: pygmt/tests: Is a directory
```

The Jupyter docummentation can be found at https://jupyter-notebook.readthedocs.io/en/stable/notebook.html?highlight=jupyter.

File: README.rst [hunk 2]
```diff
@@ -102,6 +102,20 @@ If you use `mamba <https://mamba.readthedocs.org/>`__::
 
 For other ways to install ``pygmt``, see `full installation instructions <https://www.pygmt.org/latest/install.html>`__.
 
+Getting started
++++++++++++++++
+
+As a starting point, you can open a Python console or a Jupyter Notebook, and try
```
Comment (michaelgrund) [inline README.rst:108]:
```suggestion
As a starting point, you can open a `Python console <https://docs.python.org/3/tutorial/interpreter.html>` or a `Jupyter Notebook <https://docs.jupyter.org/en/latest/running.html>`, and try 
```
Would link to the corresponding docs how to use the console and notebook.

File: README.rst [hunk 3]
```diff
@@ -102,6 +102,20 @@ If you use `mamba <https://mamba.readthedocs.org/>`__::
 
 For other ways to install ``pygmt``, see `full installation instructions <https://www.pygmt.org/latest/install.html>`__.
 
+Getting started
++++++++++++++++
+
+As a starting point, you can open a `Python console <https://docs.python.org/3/tutorial/interpreter.html>`__
```
Comment (seisman) [inline README.rst:108]:
```suggestion
As a starting point, you can open a `Python interpreter <https://docs.python.org/3/tutorial/interpreter.html>`__
```

File: README.rst (hunk 1 above)
Comment (seisman) [reply README.rst]: It seems the Jupyter documentation also inconsistently use "notebook" and "Notebook". I don't have a strong preference on either, but it seems changing "Notebook" to "notebook" is easier.

File: README.rst [hunk 4]
```diff
@@ -102,6 +102,20 @@ If you use `mamba <https://mamba.readthedocs.org/>`__::
 
 For other ways to install ``pygmt``, see `full installation instructions <https://www.pygmt.org/latest/install.html>`__.
 
+Getting started
++++++++++++++++
+
+As a starting point, you can open a `Python interpreter <https://docs.python.org/3/tutorial/interpreter.html>`__
+or a `Jupyter Notebook <https://docs.jupyter.org/en/latest/running.html>`__, and try the following example::
```
Comment (seisman) [inline README.rst:109]:
```suggestion
or a `Jupyter notebook <https://docs.jupyter.org/en/latest/running.html>`__, and try the following example::
```

File: README.rst [hunk 5]
```diff
@@ -102,6 +102,20 @@ If you use `mamba <https://mamba.readthedocs.org/>`__::
...
+
+    import pygmt
+    fig = pygmt.Figure()
+    fig.coast(projection="H10c", region="g", frame=True, land="gray")
+    fig.show()
+
+For more examples, please have a look at the `PyGMT Gallery <https://www.pygmt.org/latest/gallery/index.html>`__
```
[the remark this answers is not in the document]
Comment (yvonnefroehlich) [reply README.rst]: I agree to this change. Please see commit https://github.com/GenericMappingTools/pygmt/pull/2369/commits/89e15a75cb0a60371b7bcda7d83692d5283fe35a.

File: README.rst (hunk 1 above)
Comment (yvonnefroehlich) [reply README.rst]: The lower-case version "Jupyter notebook" is fine with me.

**Review by seisman — APPROVED** [review]

**Review by michaelgrund — APPROVED** [review]

**Pull request merged**